tests/Stripe/CollectionTest.php

```diff
@@ -19,7 +19,7 @@ final class CollectionTest extends \PHPUnit\Framework\TestCase
     public function setUpFixture()
     {
         $this->fixture = Collection::constructFrom([
-            'data' => [['id' => 1]],
+            'data' => [['id' => '1']],
```

Comment:
Updated this test to use strings instead of integers for IDs, as Stripe resource IDs should always be strings.